File: IdeaProjects/CodingQuestions/src/InterviewQuestions/Pyramid.java
package InterviewQuestions;

import java.util.Scanner;

public class Pyramid {
    public static void main(String[] args) {
//        int rows = 25; // Number of rows in the pyramid

        Scanner scanner = new Scanner(System.in);
        System.out.println("Print the number");
        int num = scanner.nextInt();


        for (int i = 1; i <= num; i++) {
            // Print spaces before the stars
            for (int j = 1; j <= num - i; j++) {
                System.out.print(" ");
            }

            // Print stars
            for (int k = 1; k <= 2 * i - 1; k++) {
                System.out.print("*");
            }

            // Move to the next line
            System.out.println();

        }
}}

File: UserInput/Pyramid.py
def print_pyramid(rows):
    """
    This function prints a pyramid pattern with the given number of rows.

    :param rows: The number of rows for the pyramid
    """
    for i in range(rows):
        # Print leading spaces
        for j in range(rows - i - 1):
            print(" ", end="")

        # Print stars
        for k in range(2 * i + 1):
            print("*", end="")

        # Move to the next line after printing each row
        print()


# Example usage
num_rows = int(input("Enter the number of rows for the pyramid: "))
print_pyramid(num_rows)
